lib/components/narrative/loading.tsx

```diff
@@ -17,4 +17,13 @@ const Loading = ({ small }: Props): JSX.Element => {
   )
 }
 
+const InlineLoading = (): JSX.Element => {
+  return (
+    <StyledIconWrapper spin>
```

Comment:
The spinning creates a disturbing effect with the focus indicator on hover:

![Screen Recording 2023-01-20 at 10 15 33 AM](https://user-images.githubusercontent.com/56846598/213733937-f870779c-7af3-484e-8eb5-45ce9f08d590.gif)

Reply:
I'm unable to replicate this, both by tabbing to the button and clicking it while holding down the mouse. How did you get this to happen?